src/aui/tabart.cpp

```diff
@@ -727,7 +727,7 @@ void wxAuiGenericTabArt::DrawButton(wxDC& dc,
     {
         case wxAUI_BUTTON_CLOSE:
             if (button_state & wxAUI_BUTTON_STATE_DISABLED)
-                bmp = m_disabledCloseBmp;
+                bmp = m_inactiveCloseBmp;
```

Comment:
Here this bitmap really stands for disabled and "active" is "normal".

So it looks that by fixing this in the other places, we break it here...

Reply:
Oh. Sorry. I missed that.

Reply:
Isn't it wrongly called `wxAUI_BUTTON_STATE_DISABLED`? In what situations can this button be disabled?

Reply:
I don't know this code, so I can't answer this immediately. It's possible that these buttons can be never disabled, but in this case this check should presumably be removed completely.

Doing this definitely doesn't seem to have high priority however...